.devcontainer/devcontainer.json

```diff
@@ -1,13 +1,17 @@
 {
-	"name": "Open PineBuds Pro - VS Code Development Environment",
+	"name": "PineBuds Pro - VS Code Dev",
```

Comment:
I think this could be adjusted to say 'Open PineBuds Pro - Devcontainer Environment', given there's Codespaces, Coder, Gitpod, and JetBrains as well that support the Devcontainer standard. It's not specifically for VS Code.

Reply:
Main reason I changed this at all is because VS Code displays it a LOT, and it was an excessively long name. I think if it has further changes, it should remain as short as possible, perhaps just "OpenPineBuds", as its fairly self explanatory that its a development/devcontainer environment.

Reply:
Could we keep it as 'OpenPineBuds - Dev Environment'?